### PR TITLE
fix(stream): prevent unfortunate line wrapping in stream

### PR DESCRIPTION
**Pull request by Chrissy**
issue-stream-2

fixes [APP-345](https://getsentry.atlassian.net/browse/APP-345?atlOrigin=eyJpIjoiYzljZmViYTEwZmJmNDZhZTk2YjkxYmI5MjczMjk5NDQiLCJwIjoiaiJ9)

### CODE REVIEW

File: src/sentry/static/sentry/app/components/eventOrGroupExtraDetails.jsx [hunk 1]
```diff
@@ -49,45 +52,44 @@ const EventOrGroupExtraDetails = createReactClass({
...
-            <TimeSince
-              date={firstSeen}
-              suffix={t('old')}
-              className="hidden-xs hidden-sm"
-            />
-          )}
-        </Flex>
+        {shortId && <GroupShortId shortId={shortId} />}
+        <Times>
+          <div css={overflowEllipsis}>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/eventOrGroupExtraDetails.jsx]: For a future PR, it'd be nice if we combined this with `TextOverflow` so we can have a component for this. `TextOverflow` is used in about 10 diff places.

**Review by billyvg — APPROVED** [review]
v unfortunate

**Comment by JTCunning** [comment]
I can't weigh in on the code but this Before/After gif is amazing.

File: src/sentry/static/sentry/app/components/eventOrGroupExtraDetails.jsx [hunk 2]
```diff
@@ -137,14 +141,35 @@ const GroupExtraCommentsAndLogger = styled(Flex)`
...
+const CommentsLink = styled(Link)`
+  display: flex;
+  align-items: center;
+  flex-shrink: 0;
+`;
+
+const Times = styled('div')`
+  margin-right: ${space(2)};
+  flex-shrink: 1;
+  min-width: 0; /* flex-hack for overflow-ellipsed children */
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/eventOrGroupExtraDetails.jsx]: I love this flex-box trick :clap:

Comment (Chrissy) [reply src/sentry/static/sentry/app/components/eventOrGroupExtraDetails.jsx]: right? 🙄

**Pull request merged**